Arm/MemoryProofs.lean

```diff
@@ -5,6 +5,9 @@ Author(s): Shilpi Goel
 -/
 import Arm.SeparateProofs
 import Arm.FromMathlib
+import Arm.State
+import Arm.Separate
+import Arm.SeparateProofs
```

Comment:
Redundant import? Maybe kill the one on Line 6 instead.

Reply:
resolved.